app/scripts/compose_json_files.py

```diff
@@ -0,0 +1,35 @@
+import json
+import os
+
+
+def create_list(input_name, subfolders = []):
+    input_list = []
+    dirpath = app_path + input_name + '/'
+    for file_name in os.listdir(dirpath):
+        if os.path.isfile(os.path.join(dirpath, file_name)):
+            json_file = json.load(open(dirpath + file_name, encoding='utf-8'))
+            input_list.append(json_file)
+    
+    
+    for subf in subfolders:
```

Comment:
This is probably fine as it is, but I thought I should mention [os.walk](https://docs.python.org/3/library/os.html#os.walk) as an alternative.